src/vterm/screen.c

```diff
@@ -2,9 +2,11 @@
 
 #include <stdio.h>
 #include <string.h>
+#include "nvim/mbyte.h"
+#include "nvim/tui/termkey.h"
 
 #include "rect.h"
-#include "utf8.h"
+#undef small
```

Comment:
Isn't that already `undef`ed in our `win_defs.h`?

Reply:
Yeah, but I assume the problematic windows header is being included somewhere else. I suspect libuv. I'll just rename all variables named "small" to "little" or something instead, this is annoying to guard against.

Reply:
"tiny"

Reply:
OK, found a workaround. `WIN32_LEAN_AND_MEAN` only includes core set of headers and not kitchen sink, so if CI passes with that let's go with that.